eclass/cmake.eclass

```diff
@@ -681,6 +681,19 @@ cmake-utils_src_make() {
 	die "cmake-utils_src_make is banned. Use cmake_build instead"
 }
 
+# @VARIABLE: CTEST_JOBS
+# @USER_VARIABLE
+# @DESCRIPTION:
+# Maximum number of CTest jobs to run in parallel.  If unset, the value
+# will be determined from make options.
+
+# @VARIABLE: CTEST_LOADAVG
+# @USER_VARIABLE
+# @DESCRIPTION:
+# Maximum load, over which no new jobs will be started by CTest.  Note
+# that unlike make, CTest will not start *any* jobs if the load
+# is exceeded.  If unset, the value will be determined from make options.
```

Comment:
Any clue what pkgcheck is complaiing about?

```
eclass
  EclassDocError: cmake: failed parsing eclass docs: '@VARIABLE:', line 684: non-inline arg
  EclassDocError: cmake: failed parsing eclass docs: '@VARIABLE:', line 690: non-inline arg

```

Reply:
Error message is not very good but, looking at pkgcore, it only accepts `@USER_VARIABLE` with `@ECLASS_VARIABLE` (not `@VARIABLE`)

Reply:
On that note, EPYTEST_JOBS is `ECLASS_VARIABLE` too.

Reply:
I followed the existing `@VARIABLE` in this eclass, but yeah, I suppose I should change that.